src/assets/simp-adapter/src/sbin/simp_rpm_helper

```diff
@@ -312,10 +311,13 @@ unless is_managed?(File.join(options.target_dir, options.module_name)) || !optio
 
   # A regular installation or upgrade
   if options.rpm_section == 'post'
-    if options.rpm_status == '2'
-      # Short circuit on upgrading safe modules, just don't touch them!
-      if safe_modules.include?(options.module_name)
-        if File.directory?(File.join(options.target_dir,options.module_name))
+    if safe_modules.include?(options.module_name)
+      # Make sure that we preserve anything in the safe modules on installation
+      options.preserve = true
+
+      if options.rpm_status == '2'
+        # Short circuit on upgrading safe modules, just don't touch them!
+        if File.directory?(File.join(options.target_dir, options.module_name))
```

Comment:
Is there any benefit in logging a message to let user know this module is being skipped?